[2022-02-10 12:35:13,108] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: spark_first_run.spark_job 2022-02-10T12:35:11.353750+00:00 [queued]>
[2022-02-10 12:35:13,154] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: spark_first_run.spark_job 2022-02-10T12:35:11.353750+00:00 [queued]>
[2022-02-10 12:35:13,155] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:35:13,155] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-02-10 12:35:13,155] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:35:13,190] {taskinstance.py:1107} INFO - Executing <Task(SparkSubmitOperator): spark_job> on 2022-02-10T12:35:11.353750+00:00
[2022-02-10 12:35:13,197] {standard_task_runner.py:52} INFO - Started process 89 to run task
[2022-02-10 12:35:13,205] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'spark_first_run', 'spark_job', '2022-02-10T12:35:11.353750+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmpyt8i331n', '--error-file', '/tmp/tmp_vvdxkky']
[2022-02-10 12:35:13,206] {standard_task_runner.py:77} INFO - Job 4: Subtask spark_job
[2022-02-10 12:35:13,303] {logging_mixin.py:104} INFO - Running <TaskInstance: spark_first_run.spark_job 2022-02-10T12:35:11.353750+00:00 [running]> on host af54f92459ec
[2022-02-10 12:35:13,437] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=spark_first_run
AIRFLOW_CTX_TASK_ID=spark_job
AIRFLOW_CTX_EXECUTION_DATE=2022-02-10T12:35:11.353750+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-10T12:35:11.353750+00:00
[2022-02-10 12:35:13,454] {base.py:69} INFO - Using connection to: id: spark_default. Host: spark://spark, Port: 7070, Schema: , Login: ***, Password: ***, extra: {'queue': 'root.default'}
[2022-02-10 12:35:13,457] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7070 --conf spark.master=spark://spark:7077 --name test --verbose --queue root.default /opt/bitnami/spark/app/test.py
[2022-02-10 12:35:17,788] {spark_submit.py:514} INFO - Using properties file: null
[2022-02-10 12:35:18,209] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2022-02-10 12:35:18,210] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.8/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.1.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2022-02-10 12:35:18,210] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2022-02-10 12:35:18,210] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2022-02-10 12:35:18,210] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2022-02-10 12:35:18,356] {spark_submit.py:514} INFO - Parsed arguments:
[2022-02-10 12:35:18,356] {spark_submit.py:514} INFO - master                  spark://spark:7070
[2022-02-10 12:35:18,356] {spark_submit.py:514} INFO - deployMode              null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - executorMemory          null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - executorCores           null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - totalExecutorCores      null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - propertiesFile          null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - driverMemory            null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - driverCores             null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2022-02-10 12:35:18,357] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2022-02-10 12:35:18,358] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2022-02-10 12:35:18,358] {spark_submit.py:514} INFO - supervise               false
[2022-02-10 12:35:18,358] {spark_submit.py:514} INFO - queue                   root.default
[2022-02-10 12:35:18,358] {spark_submit.py:514} INFO - numExecutors            null
[2022-02-10 12:35:18,358] {spark_submit.py:514} INFO - files                   null
[2022-02-10 12:35:18,358] {spark_submit.py:514} INFO - pyFiles                 null
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - archives                null
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - mainClass               null
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - primaryResource         file:/opt/bitnami/spark/app/test.py
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - name                    test
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - childArgs               []
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - jars                    null
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - packages                null
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - packagesExclusions      null
[2022-02-10 12:35:18,359] {spark_submit.py:514} INFO - repositories            null
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - verbose                 true
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - 
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - 
[2022-02-10 12:35:18,360] {spark_submit.py:514} INFO - 
[2022-02-10 12:35:19,839] {spark_submit.py:514} INFO - Main class:
[2022-02-10 12:35:19,841] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2022-02-10 12:35:19,846] {spark_submit.py:514} INFO - Arguments:
[2022-02-10 12:35:19,846] {spark_submit.py:514} INFO - file:/opt/bitnami/spark/app/test.py
[2022-02-10 12:35:19,847] {spark_submit.py:514} INFO - null
[2022-02-10 12:35:19,857] {spark_submit.py:514} INFO - Spark config:
[2022-02-10 12:35:19,858] {spark_submit.py:514} INFO - (spark.app.name,test)
[2022-02-10 12:35:19,858] {spark_submit.py:514} INFO - (spark.master,spark://spark:7070)
[2022-02-10 12:35:19,858] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2022-02-10 12:35:19,859] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2022-02-10 12:35:19,859] {spark_submit.py:514} INFO - Classpath elements:
[2022-02-10 12:35:19,860] {spark_submit.py:514} INFO - 
[2022-02-10 12:35:19,860] {spark_submit.py:514} INFO - 
[2022-02-10 12:35:19,860] {spark_submit.py:514} INFO - 
[2022-02-10 12:35:27,239] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2022-02-10 12:35:27,320] {spark_submit.py:514} INFO - 22/02/10 12:35:27 INFO SparkContext: Running Spark version 3.2.1
[2022-02-10 12:35:28,165] {spark_submit.py:514} INFO - 22/02/10 12:35:28 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2022-02-10 12:35:28,949] {spark_submit.py:514} INFO - 22/02/10 12:35:28 INFO ResourceUtils: ==============================================================
[2022-02-10 12:35:28,950] {spark_submit.py:514} INFO - 22/02/10 12:35:28 INFO ResourceUtils: No custom resources configured for spark.driver.
[2022-02-10 12:35:28,951] {spark_submit.py:514} INFO - 22/02/10 12:35:28 INFO ResourceUtils: ==============================================================
[2022-02-10 12:35:28,953] {spark_submit.py:514} INFO - 22/02/10 12:35:28 INFO SparkContext: Submitted application: test
[2022-02-10 12:35:29,156] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2022-02-10 12:35:29,196] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO ResourceProfile: Limiting resource is cpu
[2022-02-10 12:35:29,198] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2022-02-10 12:35:29,569] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO SecurityManager: Changing view acls to: default
[2022-02-10 12:35:29,570] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO SecurityManager: Changing modify acls to: default
[2022-02-10 12:35:29,571] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO SecurityManager: Changing view acls groups to:
[2022-02-10 12:35:29,572] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO SecurityManager: Changing modify acls groups to:
[2022-02-10 12:35:29,573] {spark_submit.py:514} INFO - 22/02/10 12:35:29 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(default); groups with view permissions: Set(); users  with modify permissions: Set(default); groups with modify permissions: Set()
[2022-02-10 12:35:32,240] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO Utils: Successfully started service 'sparkDriver' on port 43061.
[2022-02-10 12:35:32,442] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO SparkEnv: Registering MapOutputTracker
[2022-02-10 12:35:32,618] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO SparkEnv: Registering BlockManagerMaster
[2022-02-10 12:35:32,721] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2022-02-10 12:35:32,723] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2022-02-10 12:35:32,740] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2022-02-10 12:35:32,882] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-9fe24808-f064-40ed-83c9-2229c0feb793
[2022-02-10 12:35:33,000] {spark_submit.py:514} INFO - 22/02/10 12:35:32 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2022-02-10 12:35:33,565] {spark_submit.py:514} INFO - 22/02/10 12:35:33 INFO SparkEnv: Registering OutputCommitCoordinator
[2022-02-10 12:35:35,477] {spark_submit.py:514} INFO - 22/02/10 12:35:35 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2022-02-10 12:35:35,899] {spark_submit.py:514} INFO - 22/02/10 12:35:35 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://af54f92459ec:4040
[2022-02-10 12:35:38,663] {spark_submit.py:514} INFO - 22/02/10 12:35:38 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7070...
[2022-02-10 12:35:39,085] {spark_submit.py:514} INFO - 22/02/10 12:35:39 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7070
[2022-02-10 12:35:39,085] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2022-02-10 12:35:39,085] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2022-02-10 12:35:39,086] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2022-02-10 12:35:39,086] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2022-02-10 12:35:39,086] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2022-02-10 12:35:39,086] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2022-02-10 12:35:39,087] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2022-02-10 12:35:39,087] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-02-10 12:35:39,087] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-02-10 12:35:39,088] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-02-10 12:35:39,088] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:35:39,088] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark/172.18.0.2:7070
[2022-02-10 12:35:39,088] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2022-02-10 12:35:39,089] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2022-02-10 12:35:39,089] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2022-02-10 12:35:39,089] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2022-02-10 12:35:39,089] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2022-02-10 12:35:39,090] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2022-02-10 12:35:39,090] {spark_submit.py:514} INFO - ... 4 more
[2022-02-10 12:35:39,090] {spark_submit.py:514} INFO - Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: spark/172.18.0.2:7070
[2022-02-10 12:35:39,090] {spark_submit.py:514} INFO - Caused by: java.net.ConnectException: Connection refused
[2022-02-10 12:35:39,090] {spark_submit.py:514} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2022-02-10 12:35:39,090] {spark_submit.py:514} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
[2022-02-10 12:35:39,091] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2022-02-10 12:35:39,092] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2022-02-10 12:35:39,092] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2022-02-10 12:35:39,092] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:35:58,663] {spark_submit.py:514} INFO - 22/02/10 12:35:58 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7070...
[2022-02-10 12:35:58,677] {spark_submit.py:514} INFO - 22/02/10 12:35:58 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7070
[2022-02-10 12:35:58,677] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-02-10 12:35:58,678] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark/172.18.0.2:7070
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2022-02-10 12:35:58,679] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2022-02-10 12:35:58,680] {spark_submit.py:514} INFO - ... 4 more
[2022-02-10 12:35:58,680] {spark_submit.py:514} INFO - Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: spark/172.18.0.2:7070
[2022-02-10 12:35:58,680] {spark_submit.py:514} INFO - Caused by: java.net.ConnectException: Connection refused
[2022-02-10 12:35:58,680] {spark_submit.py:514} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2022-02-10 12:35:58,680] {spark_submit.py:514} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2022-02-10 12:35:58,680] {spark_submit.py:514} INFO - at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
[2022-02-10 12:35:58,681] {spark_submit.py:514} INFO - at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
[2022-02-10 12:35:58,681] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707)
[2022-02-10 12:35:58,681] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
[2022-02-10 12:35:58,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
[2022-02-10 12:35:58,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
[2022-02-10 12:35:58,682] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2022-02-10 12:35:58,683] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2022-02-10 12:35:58,684] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2022-02-10 12:35:58,684] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:36:18,664] {spark_submit.py:514} INFO - 22/02/10 12:36:18 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7070...
[2022-02-10 12:36:18,678] {spark_submit.py:514} INFO - 22/02/10 12:36:18 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7070
[2022-02-10 12:36:18,678] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2022-02-10 12:36:18,678] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2022-02-10 12:36:18,678] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-02-10 12:36:18,679] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark/172.18.0.2:7070
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2022-02-10 12:36:18,680] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2022-02-10 12:36:18,681] {spark_submit.py:514} INFO - ... 4 more
[2022-02-10 12:36:18,681] {spark_submit.py:514} INFO - Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: spark/172.18.0.2:7070
[2022-02-10 12:36:18,681] {spark_submit.py:514} INFO - Caused by: java.net.ConnectException: Connection refused
[2022-02-10 12:36:18,681] {spark_submit.py:514} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2022-02-10 12:36:18,681] {spark_submit.py:514} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2022-02-10 12:36:18,682] {spark_submit.py:514} INFO - at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
[2022-02-10 12:36:18,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
[2022-02-10 12:36:18,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707)
[2022-02-10 12:36:18,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
[2022-02-10 12:36:18,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
[2022-02-10 12:36:18,682] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
[2022-02-10 12:36:18,683] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2022-02-10 12:36:18,683] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2022-02-10 12:36:18,683] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2022-02-10 12:36:18,683] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:36:38,665] {spark_submit.py:514} INFO - 22/02/10 12:36:38 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2022-02-10 12:36:38,666] {spark_submit.py:514} INFO - 22/02/10 12:36:38 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2022-02-10 12:36:39,420] {spark_submit.py:514} INFO - 22/02/10 12:36:39 INFO SparkUI: Stopped Spark web UI at http://af54f92459ec:4040
[2022-02-10 12:36:39,908] {spark_submit.py:514} INFO - 22/02/10 12:36:39 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 40175.
[2022-02-10 12:36:39,908] {spark_submit.py:514} INFO - 22/02/10 12:36:39 INFO NettyBlockTransferService: Server created on af54f92459ec:40175
[2022-02-10 12:36:40,002] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO StandaloneSchedulerBackend: Shutting down all executors
[2022-02-10 12:36:40,007] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2022-02-10 12:36:40,439] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2022-02-10 12:36:40,506] {spark_submit.py:514} INFO - 22/02/10 12:36:40 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2022-02-10 12:36:40,594] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, af54f92459ec, 40175, None)
[2022-02-10 12:36:40,603] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO BlockManagerMasterEndpoint: Registering block manager af54f92459ec:40175 with 434.4 MiB RAM, BlockManagerId(driver, af54f92459ec, 40175, None)
[2022-02-10 12:36:40,610] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, af54f92459ec, 40175, None)
[2022-02-10 12:36:40,615] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, af54f92459ec, 40175, None)
[2022-02-10 12:36:40,984] {spark_submit.py:514} INFO - 22/02/10 12:36:40 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2022-02-10 12:36:41,206] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO MemoryStore: MemoryStore cleared
[2022-02-10 12:36:41,207] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO BlockManager: BlockManager stopped
[2022-02-10 12:36:41,331] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO BlockManagerMaster: BlockManagerMaster stopped
[2022-02-10 12:36:41,345] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2022-02-10 12:36:41,362] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO SparkContext: Successfully stopped SparkContext
[2022-02-10 12:36:41,748] {spark_submit.py:514} INFO - 22/02/10 12:36:41 ERROR SparkContext: Error initializing SparkContext.
[2022-02-10 12:36:41,749] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2022-02-10 12:36:41,749] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2022-02-10 12:36:41,749] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2022-02-10 12:36:41,749] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2022-02-10 12:36:41,749] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2022-02-10 12:36:41,750] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2022-02-10 12:36:41,750] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2022-02-10 12:36:41,750] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2022-02-10 12:36:41,750] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2022-02-10 12:36:41,750] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2022-02-10 12:36:41,750] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2022-02-10 12:36:41,751] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2022-02-10 12:36:41,751] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2022-02-10 12:36:41,751] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2022-02-10 12:36:41,751] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2022-02-10 12:36:41,751] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2022-02-10 12:36:41,751] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:36:41,752] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO SparkContext: SparkContext already stopped.
[2022-02-10 12:36:41,752] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2022-02-10 12:36:41,752] {spark_submit.py:514} INFO - File "/opt/bitnami/spark/app/test.py", line 10, in <module>
[2022-02-10 12:36:41,763] {spark_submit.py:514} INFO - sc = SparkContext()
[2022-02-10 12:36:41,764] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 146, in __init__
[2022-02-10 12:36:41,764] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2022-02-10 12:36:41,764] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2022-02-10 12:36:41,764] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/py4j-0.10.9.3-src.zip/py4j/java_gateway.py", line 1585, in __call__
[2022-02-10 12:36:41,764] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/py4j-0.10.9.3-src.zip/py4j/protocol.py", line 326, in get_return_value
[2022-02-10 12:36:41,767] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2022-02-10 12:36:41,767] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2022-02-10 12:36:41,767] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2022-02-10 12:36:41,767] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2022-02-10 12:36:41,767] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2022-02-10 12:36:41,768] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-02-10 12:36:41,769] {spark_submit.py:514} INFO - 
[2022-02-10 12:36:41,964] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO ShutdownHookManager: Shutdown hook called
[2022-02-10 12:36:41,967] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO ShutdownHookManager: Deleting directory /tmp/spark-8c61f911-adbd-4741-894a-148ad9d45057
[2022-02-10 12:36:41,970] {spark_submit.py:514} INFO - 22/02/10 12:36:41 INFO ShutdownHookManager: Deleting directory /tmp/spark-c70a0330-6b5b-4991-b869-c086f4440830
[2022-02-10 12:36:42,177] {taskinstance.py:1501} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1157, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1331, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1361, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 182, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 445, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7070 --conf spark.master=spark://spark:7077 --name test --verbose --queue root.default /opt/bitnami/spark/app/test.py. Error code is: 1.
[2022-02-10 12:36:42,254] {taskinstance.py:1544} INFO - Marking task as FAILED. dag_id=spark_first_run, task_id=spark_job, execution_date=20220210T123511, start_date=20220210T123513, end_date=20220210T123642
[2022-02-10 12:36:42,393] {local_task_job.py:151} INFO - Task exited with return code 1
